pubsublite/driver/driver.go

```diff
@@ -0,0 +1,9 @@
+package driver
+
+import "context"
+
+type Pubsubber interface {
+	Publish(ctx context.Context, topic string, msg []byte) error
+	Subscribe(ctx context.Context, topic string, callback func(msg []byte)) error
```

Comment:
How about
```
Receive(ctx, topic) (msg, error)
```

Reply:
If you really want this kind of synchronous API (I don't but I'm biased towards callbacks), I think it would make more sense to have something like this:
```
type Pubsubber interface {
    ...
    Subscribe(ctx context.Context, topic string) (Receiver, error)
}

type Receiver interface {
    Receive(ctx context.Context) (Msg, error)
}
```
The `Receiver` would keep a subscription open and do any necessary local buffering to make sure messages don't get dropped if the client temporarily fails to keep up.

To be more in keeping with "take interfaces, return concrete types", it could be 
```
type Pubsubber interface {
    ...
    Subscribe(ctx context.Context, topic string, h Handler) error
}

type Handler interface {
    Handler(ctx context.Context, Msg) error
}
```
but that's essentially a more verbose way to write the callback-based thing I suggested. Maybe people would find it easier to think about somehow.